<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertImage</class>
 <widget class="QWidget" name="ConvertImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConvertImage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="5">
    <widget class="QCheckBox" name="checkBox_isCheck_size">
     <property name="text">
      <string>尺寸归一检查</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="pushButton_Default">
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>文件列表:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>输出数据库名:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QCheckBox" name="checkBox_isGray">
     <property name="text">
      <string>转换为灰度图</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QCheckBox" name="checkBox_isShuffle">
     <property name="text">
      <string>随机调整顺序</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>图片集根目录:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="pushButton_BrowserRootDir">
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>输出目录:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="pushButton_BrowserFileList">
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>调整图片高度为:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QCheckBox" name="checkBox_isEncoded">
     <property name="text">
      <string>置入图片编码</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="pushButton_BrowserOutputDir">
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="7" column="7">
    <widget class="QPushButton" name="pushButton_DoConvert">
     <property name="text">
      <string>转换</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>调整图片宽度为:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="lineEdit_resize_width">
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEdit_RootDir"/>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEdit_ImgFileList"/>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEdit_OutputDir"/>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEdit_DBName"/>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="lineEdit_resize_height"/>
   </item>
   <item row="6" column="7">
    <widget class="QRadioButton" name="radioButton_Leveldb">
     <property name="text">
      <string>LevelDb</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QRadioButton" name="radioButton_Lmdb">
     <property name="text">
      <string>LmDb</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="lineEdit_encode_type"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>编码类型:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>进度:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_BrowserFileList</sender>
   <signal>clicked()</signal>
   <receiver>ConvertImage</receiver>
   <slot>Browser1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_BrowserRootDir</sender>
   <signal>clicked()</signal>
   <receiver>ConvertImage</receiver>
   <slot>Browser2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_BrowserOutputDir</sender>
   <signal>clicked()</signal>
   <receiver>ConvertImage</receiver>
   <slot>Browser3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Default</sender>
   <signal>clicked()</signal>
   <receiver>ConvertImage</receiver>
   <slot>SetDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_DoConvert</sender>
   <signal>clicked()</signal>
   <receiver>ConvertImage</receiver>
   <slot>DoConvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Browser1()</slot>
  <slot>Browser2()</slot>
  <slot>Browser3()</slot>
  <slot>SetDefault()</slot>
  <slot>DoConvert()</slot>
 </slots>
</ui>
